Source/Prism/Events/EventSubscription.cs

```diff
@@ -50,7 +50,7 @@ public Action Action
         /// <remarks>
         /// If <see cref="Action"/>is no longer valid because it was
         /// garbage collected, this method will return <see langword="null" />.
-        /// Otherwise it will return a delegate that evaluates the <see cref="Filter"/> and if it
+        /// Otherwise it will return a delegate that evaluates the <see cref="IDelegateReference"/> and if it
```

Comment:
I only modified it because there was a warning that it could not resolve the cref.  The filter appeared to be an instance of IDelegateReference.  However, if you say it should remain Filter, I won't argue with you since you are the maintainer of the source code and would know better than me.